projects/go-tester/src/app/components/test-page-1/test-page-1.component.html

```diff
@@ -83,19 +84,24 @@
     <go-table-column field="email" [sortable]="true" title="Email"></go-table-column>
     <go-table-column field="gender" title="Gender"></go-table-column>
     <go-table-column field="ip_address" title="IP Address"></go-table-column>
-    <ng-template #goTableRowDetails let-item>
-      <tr class="child-row" *ngFor="let childItem of item.children; let last = last">
-        <td>
-          <span class="active-child-row" [ngClass]="{ 'last-active-child': last }"></span>
-        </td>
-        <td>{{ childItem.id }}</td>
-        <td>{{ childItem.name.first }}</td>
-        <td>{{ childItem.name.last }}</td>
-        <td>{{ childItem.email }}</td>
-        <td>{{ childItem.gender }}</td>
-        <td>{{ childItem.ip_address }}</td>
-      </tr>
-    </ng-template>
+
+    <ng-container #goTableChildRows>
+      <go-table-child-column field="id"></go-table-child-column>
+      <go-table-child-column field="name.first"></go-table-child-column>
+      <go-table-child-column field="name.last"></go-table-child-column>
+      <go-table-child-column field="email"></go-table-child-column>
+      <go-table-child-column field="gender"></go-table-child-column>
+      <go-table-child-column>
+        <ng-template #goTableCell let-childItem let-parentItem="parentItem">
+          {{ childItem.ip_address }}
+          <go-icon-button
+            buttonIcon="explore"
+            (handleClick)="logRow(parentItem, childItem)">
+          </go-icon-button>
+        </ng-template>
+      </go-table-child-column>
+    </ng-container>
```

Comment:
This is what I am proposing for implementation. We specify a container for the child rows, then each row is represented by a `go-table-child-column` component in the exact way that the parent rows are defined.

In the last column, I pass a custom template and show how you could access both the current item and the parent item.